Tools/machines/lxplus-cern/spack.yaml

```
@@ -0,0 +1,62 @@
+spack:
+  # variations of this environment can be controlled via environment variables during `spack env create`
+  definitions:
+  # CUDA
+  - cuda: []
+  - cuda: [cuda, blaspp +cuda]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+
+  # basic Python dependencies for WarpX Python PICMI bindings
+  - python: []
+  - python: [python, py-pip]
```

We can also request to install the optional Python bindings of openPMD-api by default in that case:
```suggestion
  - python: [python, py-pip, openpmd-api +python]
```